theme/scripts/fadeIn.ts

```diff
@@ -0,0 +1,57 @@
+import isInViewport from './utils/isInViewport';
+import throttle from 'lodash/throttle';
+
+export default (function() {
+  const FadeIn = {
+    SELECTORS: {
+      gridItem: "[data-grid-item]",
+    },
+    ATTRIBUTES: {
+      fadeIn: "data-fade-in",
+    },
+
+    init() {
+      const gridItems = [].slice.call(
+        document.querySelectorAll(FadeIn.SELECTORS.gridItem)
+      );
+
+      if (!gridItems) {
+        return;
+      } else {
+        FadeIn.bindEventListeners();    
+      }
+    },
+
+    bindEventListeners() {
+      const gridItems = [].slice.call(
```

Comment:
These should get passed in, since you are already calling this on `init`

Reply:
Is it better to pass it in, to avoid having to "query " that selector again? 
```
      const gridItems = [].slice.call(
        document.querySelectorAll(FadeIn.SELECTORS.gridItem)
      );
```

Reply:
Yeah, just kind of redundant. Also, just good functional programming :)

Reply:
Gotchaa, thank yoU!